cypress/support/commands.js

```diff
@@ -63,19 +95,19 @@ Cypress.Commands.add('checkMenu', (table) => {
   });
 })
 
-Cypress.Commands.add('addUser', (userName, role) => {
+Cypress.Commands.add('addUser', (userName, name) => {
   const createUser = {
     TemporaryPassword: TEMP_PASSWORD,
     MessageAction: 'SUPPRESS',
     UserAttributes: [
-      { Name: NAME, Value: 'Test User' },
-      { Name: CUSTOM_IDENTIFIERS, Value: 'feide:test@unit.no' },
+      { Name: NAME, Value: name },
+      { Name: CUSTOM_IDENTIFIERS, Value: `feide:${userName}` },
       { Name: CUSTOM_ORG_LEGAL_NAME, Value: 'Unit' },
       { Name: CUSTOM_ORG_NUMBER, Value: 'NO818477822' },
       { Name: CUSTOM_APPLICATION, Value: 'NVA' },
-      { Name: CUSTOM_APPLICATION_ROLES, Value: role },
-      { Name: CUSTOM_COMMON_NAME, Value: 'Test User' },
-      { Name: CUSTOM_FEIDE_ID, Value: 'test@unit.no' },
+      { Name: CUSTOM_APPLICATION_ROLES, Value: 'Publisher' },
```

Comment:
denne kommer trolig ikke til å bli brukt fremover, men erstattet av eget endpoint for roller. men kan vente til over sommeren når roller-biten har fått modnet litt før vi gjør noe her

Reply:
Tenker det samme